bindings/java/src/junit/com/apple/foundationdb/RangeQueryTest.java

```diff
@@ -50,8 +53,8 @@ public void execute(Runnable command) {
 		}
 	};
 
-	private static FDBDatabase makeFakeDatabase(List<Map.Entry<byte[], byte[]>> data) {
-		return new FDBDatabase(1, EXECUTOR) {
```

Comment:
Java unit tests were failing in FDBDatabase constructor because it was trying to call Database_setOption